package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "scripts": {
     "start": "DEBUG=ERROR-*,WARN-*,INFO-* node build/bot",
```

Comment:
You sure you don't need to require `dotenv/config` here too?

Reply:
I was considering it, but I think for production we'll use process env vars for sure, so not needed.

It doesn't harm to add it anyways, what do you think?

Reply:
not needed then